app/controllers/user_journey_controller.rb

```diff
@@ -4,21 +4,14 @@ class UserJourneyController < ApplicationController
   include ComponentConcern
   include CertificateConcern
 
+  before_action :find_teams, :find_certificate, except: :index
```

Comment:
why are we calling the `find_teams` method before each of these actions?

Reply:
Sorry what did you mean by `this should now be restricted by the team, unless GDS user`?